tests/attacks/lucash-flap-end.mcd.expected

```diff
@@ -60,6 +60,8 @@
       ListItem ( LogNote ( ADMIN , Gem "gold" . mint "Bobby" 20 ) )
       ListItem ( LogNote ( ADMIN , Gem "MKR" . initUser "Alice" ) )
       ListItem ( LogNote ( ADMIN , Gem "MKR" . initUser "Bobby" ) )
+      ListItem ( Exception ( Gem "MKR" . initUser Flap ) )
```

Comment:
`transact ADMIN Gem "MKR" . initUser Flap` is generating an exception. Is this intended?